conda-forge.yml

```
@@ -1,5 +1,6 @@
 conda_forge_output_validation: true
-provider: {linux_aarch64: azure}
+provider:
+  linux_aarch64: azure
```

Travis is back now, let's give it a shot:
```suggestion
  linux_aarch64: default
```